src/utilities/findUserWithFullName/findUserWithFullName.test.js

```diff
@@ -0,0 +1,34 @@
+import { findUserWithFullName } from './findUserWithFullName';
+
+describe('The findUserWithFullName function', () => {
+  describe('when provided with a valid users array', () => {
+    let usersArray;
+    beforeEach(() => {
+      usersArray = [
+        {
+          firstName: 'John',
+          lastName: 'Smith',
+          heightInCm: 184,
+        },
+        {
+          firstName: 'Kate',
+          lastName: 'Williams',
+          heightInCm: 169,
+        },
+      ];
+    });
+    describe('when provided with a valid full name', () => {
```

Comment:
To be grammatically correct, you should write "and when provided with a valid full name".